uber/reports.py

```diff
@@ -0,0 +1,48 @@
+from uber.common import *
+
+
```

Comment:
yea, the other thing I would love to do is figured out how to have multiple plugins be able to get their hooks into reports like this.  so if barcode, and some other plugin, wanted to each add a column to the report, that's what I would like to see happen.  maybe some kind of array in the class that does something similar here.